docs/site/Express-middleware.md

```diff
@@ -0,0 +1,229 @@
+---
+lang: en
+title: 'Use Express middleware'
+keywords: LoopBack 4.0, LoopBack 4, Express, Middleware
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Express-middleware.html
+---
+
+## Overview
+
+Express is the most popular web framework for Node.js developers. Middleware is
+the basic building blocks for Express applications. The following is quoted from
+[Express web site](https://expressjs.com/en/guide/using-middleware.html):
+
+> Express is a routing and middleware web framework that has minimal
+> functionality of its own: An Express application is essentially a series of
+> middleware function calls.
+
+LookBack 4 leverages Express behind the scene for its REST server
+implementation. We haven't exposed middleware capabilities to users before an
+elegant way can be crafted to nicely fit Express middleware into the LoopBack 4
+programming model. We believe that we now have a good middle-ground to combine
+the best of breeds of both projects.
+
+The integration is built on top of the [interceptor](Interceptors.md) support as
+follows:
+
+We create a few helper functions that wraps an Express middleware module into an
+LoopBack 4 interceptor.
+
+### Create an interceptor from Express middleware module name or factory function
```

Comment:
Since this file will be a reference doc, we should prioritize on user API documentation. Including implementation details should be an added bonus, we could even omit it, IMO.

Reply:
We can improve it once the apidocs is live.